docs/en/sql-reference/statements/system.md

```diff
@@ -578,7 +578,7 @@ Wait for the currently running refresh to complete. If the refresh fails, throws
 
 ### STOP VIEW, STOP VIEWS {#stop-view-stop-views}
```

Comment:
nit: since we're adding new system queries, will it make sense to document them here as well?

Reply:
I didn't want to make it an official feature, it'd be confusing:
 * most users would probably expect `SYSTEM START REPLICATED VIEW` to undo the effect of `SYSTEM STOP VIEW`, but it doesn't, the per-replica pause and the global pause are two separate flags that are ORed together,
 * `SYSTEM STOP VIEW` interrupts a running refresh, but `SYSTEM STOP REPLICATED VIEW` doesn't. (Though it would be easy to fix if needed.)

The idea was to have an easy way to unbreak things if a view somehow ends up paused (e.g. backup fails in the middle, but for some reason we don't want to drop everything and start over), but to discourage the use of global-pause for anything else.